etc/profile-m-z/telegram.profile

```diff
@@ -36,10 +36,15 @@ noroot
 notv
 protocol unix,inet,inet6,netlink
 seccomp
+seccomp.block-secondary
 shell none
+tracelog
 
 disable-mnt
+#private-bin bash,sh,telegram-desktop
```

Comment:
There seem to be at least three names for the binary:

```
$ ls /etc/firejail | grep -i telegram
telegram-desktop.profile
telegram.profile
Telegram.profile
```

```suggestion
#private-bin bash,sh,telegram,Telegram,telegram-desktop
```

and bash,sh is blacklisted by `include disable-shell.inc` so a (commented) `include allow-bin-sh.inc` would make sense.

Reply:
I'm running with `include disable-shell.inc` & without `include allow-bin-sh.inc`. Wait a minute, telegram was not running without `bash` or `sh`. So with `include disable-shell.inc` telegram should not run anyway if I'm not wrong. But,I didn't face any issue whatsoever.

Reply:
well got the reason. it was getting called by `dbus`. though with `private-bin` I don't need `include allow-bin-sh.inc` or `dbus`. A bit confused here.What's your interpretation?

Reply:
I was running `telegram` from terminal without referring  absolute path. probably that's why `bash` was needed. telegram can run without shell it seems. Sorry for all the non-sense. need a good sleep.